damus/Models/ImageHost.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ImageHost.swift
+//  damus
+//
+//  Created by Michael Hall on 01/04/23.
+//
+
+import Foundation
+import UIKit
+
+enum ImageHost: String, CaseIterable, Identifiable {
+    var id: String { self.rawValue }
+    
+    struct Model: Identifiable, Hashable {
+        var id: String { self.tag }
+        var tag: String
+        var displayName : String
+        var uploadLink : String
+        var clientId : String?
+    }
+    
+// TODO: implement nostr.build
+//    case nostr_build
+    case nostrimg
+    
+    var model: Model {
+        switch self {
+// TODO: implement nostr.build
+//        case .nostr_build:
+//            return .init(tag: "nostr_build", displayName: NSLocalizedString("nostr.build", comment: "Dropdown option label for image host Nostr.Build."),
+//                         uploadLink: "https://nostr.build/api/upload", clientId: "")
+        case .nostrimg:
+            return .init(tag: "nostrimg", displayName: NSLocalizedString("nostrimg.com", comment: "Dropdown option label for image host Nostrimg."),
+                         uploadLink: "https://nostrimg.com/api/upload", clientId: "401e1e2049ba44114a5f59e60315eb0fb0fd12beee1f08deb8bde773995b896e")
```

Comment:
I do not want to rely on any hardcoded integrations like this unless it's something that I know will keep working